pkg/asset/cluster/tfvars/tfvars.go

```diff
@@ -515,6 +516,29 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			}
 		}
 
+		ctx, cancel := context.WithTimeout(context.TODO(), 60*time.Second)
+		defer cancel()
+
+		bucketName := gcpbootstrap.GetBootstrapStorageName(clusterID.InfraID)
+		bucketHandle, err := gcpbootstrap.CreateBucketHandle(ctx, bucketName)
+		if err != nil {
+			return fmt.Errorf("failed to create bucket handle %s: %w", bucketName, err)
+		}
+
+		bootstrapIgnURL, err := gcpbootstrap.ProvisionBootstrapStorage(ctx, installConfig, bucketHandle, clusterID.InfraID)
+		if err != nil {
+			return fmt.Errorf("failed to provision gcp bootstrap storage resources: %w", err)
+		}
+
+		if err := gcpbootstrap.FillBucket(ctx, bucketHandle, bootstrapIgn); err != nil {
+			return fmt.Errorf("failed to fill bootstrap ignition bucket: %w", err)
+		}
```

Comment:
if we upload the bootstrap.ign file here, won't that break the custom dns, as the ignition needs to be updated with the LB data from the terraform run? 

It looks like it might be possible to still handle the bootstrap ignition upload in terraform?

Reply:
This shouldn't break the custom DNS runs. The data that is stored in the bucket can be edited.

Reply:
Ok, it would be possible to upload it here and do another upload later with the updated bootstrap ignition. But this PR is still removing the terraform config where we upload the bootstrap.ign with the lb config in TF, so as it stands I think it will break.

I would try skipping this call to `gcpbootstrap.FillBucket` here (it seems like we can generate the shim without it) and restore the terraform resource `resource "google_storage_bucket_object" "ignition" {`